src/index.html

```diff
@@ -0,0 +1,27 @@
+---
+---
+
+<!DOCTYPE html>
+<html lang="pt">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Saira&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
why is saira's import over here but orbitron's import over the main .scss file? choose one.

Reply:
I choose the main .scss file